### ISSUE TITLE
Debugger to support $ReturnValue

### ISSUE THREAD

**Issue by jchannon** [OmniSharp/omnisharp-vscode#859]
## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (1.0.0-preview2-003131)

Product Information:
 Version:            1.0.0-preview2-003131
 Commit SHA-1 hash:  635cf40e58

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.11
 OS Platform: Darwin
 RID:         osx.10.11-x64

```
VS Code version:1.6.1
C# Extension version:1.4.1

## Steps to reproduce
Debug a method that returns a statement eg/ `return Foo();`

## Expected  behavior
Debugger watch command to show value returned via `$ReturnValue` value

## Actual behavior
`$ReturnValue` not defined.

`$ReturnValue` is a variable available in Visual Studio that shows returned values from methods in the immediate/watch window. Would be nice to have the same in VSCode

**Comment by gregg-miskelly** [comment]
Hmm... I am not quite sure why this doesn't work. We will have to look into this.

**Comment by code1line** [comment]
Any news on this issue?

**Comment by lijunle** [comment]
Still an issue on new year!

**Comment by gregg-miskelly** [comment]
Fixed in 1.21

**Comment by gregg-miskelly** [comment]
Sorry, I think this was actually fixed in 1.18, but was incorrectly left open.
